src/CarwowTheme/Modal.elm

```diff
@@ -51,10 +51,10 @@ type Msg
 
 {-| Placeholder
 -}
-init : String -> Model
-init id =
+init : String -> Bool -> Model
+init id isOpen =
     { id = id
-    , isOpen = False
+    , isOpen = isOpen
```

Comment:
Is this being used somewhere? Are there some accompanying PRs for where this is being consumed from?

Reply:
It's not being used yet.
I’m looking to add direct navigation to the MakeModelMenu using hash links and need to have the option to init the Modal being open.

Reply:
@muyiwaolu e.g this PR: https://github.com/carwow/elm-core/pull/15